assets/asset/Player.png.import

```diff
@@ -3,15 +3,15 @@
 importer="texture"
 type="CompressedTexture2D"
 uid="uid://bjhqsrrjydd46"
-path="res://.godot/imported/Player.png-8c45d00b25ed1c2ee3b4c902405c7bf6.ctex"
+path="res://.godot/imported/Player.png-833f18bdce549377c0230f8fa0b76e96.ctex"
 metadata={
 "vram_texture": false
 }
 
 [deps]
 
-source_file="res://scenes/combat/asset/Player.png"
-dest_files=["res://.godot/imported/Player.png-8c45d00b25ed1c2ee3b4c902405c7bf6.ctex"]
+source_file="res://assets/asset/Player.png"
```

Comment:
Good move into the assets folder, took me a while to find it deep in the scenes folder